core-chart/values.yaml

```diff
@@ -58,8 +59,20 @@ InstallPCHs: true
 
 # Transport controller bundling parameters
```

Comment:
It's not just _bundling_ parameters any more.

Reply:
I changed to just "parameters"